src/main/Main.java

```diff
@@ -25,6 +24,9 @@
         // main frame panel setting
         this.add(this.start);
         this.setVisible(true);
+        //test sound
+        new Sound("start");
+        
```

Comment:
Start sound only has few second, but the user’s operating life cycle is unpredictable, I think to find the music which can loop will be better